test/trento_web/controllers/tags_controller_test.exs

```diff
@@ -18,13 +18,26 @@ defmodule TrentoWeb.TagsControllerTest do
                }
              } = json_response(conn, 400)
     end
+
+    test "should decline tag with forbidden characters", %{conn: conn} do
+      conn =
+        post(conn, Routes.hosts_tagging_path(conn, :add_tag, Faker.UUID.v4()), %{
+          "value" => "This / is a \ wrong #tag"
+        })
+
+      assert %{
+               "errors" => %{
+                 "value" => ["has invalid format"]
+               }
+             } = json_response(conn, 400)
+    end
   end
 
   describe "tagging sap systems and databases" do
     test "should add a tag to a sap system", %{conn: conn} do
       conn =
         post(conn, Routes.sap_systems_tagging_path(conn, :add_tag, Faker.UUID.v4()), %{
-          "value" => Faker.Beer.style()
+          "value" => String.replace(Faker.Beer.style(), " ", "")
```

Comment:
I'd propose to switch `Faker.Beer` with something that maybe doesn't have spaces, wdyt?

There is e.g. https://github.com/elixirs/faker/blob/master/lib/faker/color/en.ex
(much more boring ofc... ;-p)

Reply:
Haha that's true... well I will miss the beer, but you are right thanks :D